time_tracker_api/__init__.py

```diff
@@ -34,16 +35,18 @@ def init_app_config(app: Flask, config_path: str, config_data: dict = None):
 
 
 def init_app(app: Flask):
-    from .database import init_app as init_database
+    from time_tracker_api.database import init_app as init_database
     init_database(app)
 
-    from .api import api
+    from time_tracker_api.api import api
     api.init_app(app)
 
     if app.config.get('DEBUG'):
+        app.logger.setLevel(logging.INFO)
```

Comment:
should not be `app.logger.setLevel(logging.DEBUG)`?

Reply:
`logging.INFO` is too detailed.